src/util.ts

```diff
@@ -150,3 +152,12 @@ export const fileExists = async (path: string) => {
         return false;
     }
 };
+
+// We can't use `runAndroidDevTool` for this because that can only give you the process if you `await` it, which we
+// don't want.
+export const startEmulator = async (name: string, args: string[]) => {
+    const { env } = await ensureSdkmanager();
+    const toolPath = await getAndroidDevToolPath('emulator');
+
+    return () => execa(toolPath, ['-avd', name, ...args], { env, stdio: 'pipe' });
```

Comment:
I am not sure why you set this to `pipe`, but I think the output would be useful:

```suggestion
    return () => execa(toolPath, ['-avd', name, ...args], { env, stdio: 'inherit' });
```

Reply:
Oops, that is left over from debugging.

Reply:
The default is `pipe`, so removing it does not show the output either.

Reply:
I don't want to show the output…

Reply:
But that does explain why I didn't see any output during debugging. :D